R/graded_condition.R

```diff
@@ -15,11 +15,18 @@ conditionMessage.gradethis_graded <- function(c) {
   } else {
     "[Neutral]"
   }
-
-  paste(
-    correct,
-    paste(as.character(condition_obj$message), collapse = " ")
-  )
+  
+  message <- paste(as.character(condition_obj$message), collapse = " ")
+  len_message <- nchar(message) + nchar(correct) + 3 + nchar("<gradethis_graded>")
```

Comment:
Should `len_message` account for internal `\n` in `message`?

Reply:
Yeah, I'll have those messages wrap regardless if a newline is detected